log_transfer/tests/test_audit_logging.py

```diff
@@ -247,34 +296,82 @@ def test_send_timestamp_in_root(user, fixed_datetime):
 
 
 @pytest.mark.django_db
-@override_settings(CLEAR_AUDIT_LOG_ENTRIES=True)
-def test_clear_audit_log(user, fixed_datetime):
+@override_settings(
+    USE_DJANGO_AUDITLOG=False,
+    CLEAR_AUDIT_LOG_ENTRIES=True,
+)
+def test_clear_audit_log(user, fixed_datetime, settings):
     addresses = ["192.168.1.1", "192.168.1.2", "192.168.1.3"]
     for addr in addresses:
         audit_logging.log(
-          user,
-          "shared.oidc.auth.HelsinkiOIDCAuthenticationBackend",
-          Operation.READ,
-          user,
-          get_time=fixed_datetime,
-          ip_address=addr
+            user,
+            "shared.oidc.auth.HelsinkiOIDCAuthenticationBackend",
+            Operation.READ,
+            user,
+            get_time=fixed_datetime,
+            ip_address=addr,
```

Comment:
Unnecessary comma added?

Reply:
Trailing commas lead to smaller diffs when making changes, so I generally add them when they are missing, but I can remove them if you'd like.

Reply:
Ok, good point, let's keep your change as is for this comment.